.github/PULL_REQUEST_TEMPLATE/app_update_template.md

```diff
@@ -0,0 +1,40 @@
+<!-- 
+Thank you for submitting an update for an existing app on the Umbrel App Store! 
+Please fill out all sections below to help us review and merge your update smoothly.
+-->
+
+# App Update Submission
+
+### App name *(required)*
+<!-- Name of the app being updated -->
+
+### Current app version on Umbrel *(required)*
+<!-- Version currently listed in the Umbrel App Store -->
+<!-- Example: 1.5.0 -->
+
+### New app version *(required)*
+<!-- Version you are updating to -->
+<!-- Example: 1.6.0 -->
+
+### Changelog / Release Notes *(required)*
+<!-- Briefly summarize what's new, fixed, or changed in this update -->
+<!-- You can link to the official upstream changelog if available -->
+<!-- Example:
+- Updated to Nextcloud 1.6.0
+- Fixed login redirect issue
+- Added Redis cache support
+-->
+
+### I have tested my app on *(check at least one)*:
+* [ ]  umbrelOS on a Raspberry Pi
+* [ ]  umbrelOS on an Umbrel Home
+* [ ]  umbrelOS on Linux VM
+
+### App architecture support *(required)*
+<!-- Confirm which architectures this update supports -->
+* [ ]  amd64
+* [ ]  arm64
```

Comment:
Not sure if this is needed as the linter github actions checks against this already.